distributed/scheduler.py

```diff
@@ -4183,7 +4171,7 @@ async def remove_worker(self, address, stimulus_id, safe=False, close=True):
         logger.info("Remove worker %s", ws)
         if close:
             with suppress(AttributeError, CommClosedError):
-                self.stream_comms[address].send({"op": "close", "report": False})
+                self.stream_comms[address].send({"op": "close"})
```

Comment:
⚠️ This will now cause workers with nannies to _restart_, not shut down. `Scheduler.remove_worker` is called in many places. You could argue that most of them warrant a restart, since it's in response to a comm error or something, but are we sure about that?

Here at least, it doesn't (though this case should be changed anyway https://github.com/dask/distributed/issues/6227): https://github.com/dask/distributed/blob/33fc50ca9817216bb4105b68f5e0859ebfb80fdb/distributed/scheduler.py#L6127-L6130